### PR TITLE
PB-397 Simplify write metrics function

### PR DESCRIPTION
**Pull request by Robert-Steiner**
### References

[PB-397](https://xainag.atlassian.net/browse/PB-397)

### Summary

* removed metrics in round class
* add jsonschema to validate the metrics JSON object
* updated tests
* simplified write metrics function

### Are there any open tasks/blockers for the ticket?

No

---

### Reviewer checklist

*Reviewer agreement:*

* Reviewers assign themselves at the start of the review.
* Reviewers do **not** commit or merge the merge request.
* Reviewers have to check and mark items in the checklist.

**Merge request checklist**

- [x] Conforms to the merge request title naming `XP-XXX <a description in imperative form>`.
- [x] Each commit conforms to the naming convention `XP-XXX <a description in imperative form>`.
- [x] Linked the ticket in the merge request title or the references section.
- [x] Added an informative merge request summary.

**Code checklist**

- [x] Conforms to the branch naming `XP-XXX-<a_small_stub>`.
- [x] Passed scope checks.
- [x] Added or updated tests if needed.
- [x] Added or updated code documentation if needed.
- [x] Conforms to Google docstring style.
- [x] Conforms to XAIN structlog style.

### CODE REVIEW

File: tests/test_metric_store.py [hunk 1]
```diff
@@ -1,72 +1,101 @@
...
+    return json.dumps([])
 
 
-def test_transform_data(metrics_sample):
-    """Check that a metric object is correctly transformed into the influx data point structure."""
-    actual_data_points = transform_metrics_to_influx_data_points(
-        "participant_id", metrics_sample
+@pytest.fixture()
+def metrics_sample_invalid():
+    """Return a valid metric object."""
```
Comment (janpetschexain) [inline tests/test_metric_store.py]: `invalid` instead of `valid`

File: tests/test_metric_store.py [hunk 2]
```diff
@@ -1,72 +1,101 @@
...
-
-    assert actual_data_points[0]["time"] == actual_data_points[2]["time"]
-    assert actual_data_points[1]["time"] == actual_data_points[3]["time"]
+
+
+@mock.patch.object(InfluxDBClient, "write_points", return_value=True)
+def test_valid_metric(
+    write_points_mock, metrics_sample,
+):  # pylint: disable=redefined-outer-name,unused-argument
+    """Check that raised exceptions of the write_points method are caught in the write_metrics
```
Comment (janpetschexain) [inline tests/test_metric_store.py]: this description doesn't match the content of the test

File: tests/test_metric_store.py (hunk 1 above)
Comment (Robert-Steiner) [reply tests/test_metric_store.py]: Fixed

File: tests/test_metric_store.py (hunk 2 above)
Comment (Robert-Steiner) [reply tests/test_metric_store.py]: Fixed

**Pull request merged**